scripts/azure-pipelines/linux-check.sh

```diff
@@ -33,11 +33,18 @@ if [[ "$DO_COVERAGE_CHECK" == "yes" ]] ; then
     cmake_args="$cmake_args -DENABLE_COVERAGE=ON -DGCOV_EXECUTABLE=/home/serac/gcov"
 fi
 
+if [[ "$DO_STYLE_CHECK" == "yes" ]] ; then
+    cmake_args="$cmake_args -DENABLE_CLANGFORMAT=ON -DCLANGFORMAT_EXECUTABLE=/usr/bin/clang-format"
+    echo "~~~~~ START: clang executables ~~~~~"
+    ls -al /usr/bin | grep clang
+    echo "~~~~~ END: clang executables ~~~~~"
```

Comment:
Was this debugging snippet intentionally left in?

Reply:
In my late night coding it seemed like a good idea at the time.  removed for a more targeted erroring out.